tutorials/scripting/gdscript/gdscript_basics.rst

```diff
@@ -338,17 +338,24 @@ For instance, you can use it to export a value to the editor::
 For more information about exporting properties, read the :ref:`GDScript exports <doc_gdscript_exports>`
 article.
 
+Any constant expression can be passed as an annotation argument::
```

Comment:
```suggestion
Any constant expression that is compatible with the required argument type can be passed as an annotation argument value::
```

Reply:
I would remove `that is` . But I'm fine with each of the suggested sentences.

Reply:
Done.